frontend-v2/src/features/simulation/SimulationSliderView.tsx

```diff
@@ -231,8 +231,10 @@ const SimulationSliderView: FC<SimulationSliderProps> = ({
           min={minValue}
           max={maxValue}
           step={stepValue}
+          shiftStep={stepValue * 10}
           onChange={handleSliderChange}
-          onChangeCommitted={handleBlur}
```

Comment:
`onChangeCommitted` works for pointer events, but commits changes on every slider step when you use the keyboard. 🤷‍♂️